sdk-java/src/main/java/io/littlehorse/sdk/wfsdk/internal/WorkflowImpl.java

```diff
@@ -55,6 +57,10 @@ public void addExternalEventDefName(String eedName) {
         requiredEedNames.add(eedName);
     }
 
+    public void addWorkflowEventDefName(String name) {
```

Comment:
This should be private

Reply:
It cannot be. That is because the `WorkflowThreadImpl` has to call it. Also, note that this is the `Impl` not the `Workflow`.

See `addTaskDefName`

Reply:
I meant package private

Reply:
Package private is possible. Doing that.